array_connection.go

```diff
@@ -119,13 +151,35 @@ func CursorForObjectInConnection(data []interface{}, object interface{}) Connect
 	return offsetToCursor(offset)
 }
 
-func getOffset(cursor ConnectionCursor, defaultOffset int) int {
+func getOffsetWithDefault(cursor ConnectionCursor, defaultOffset int) int {
```

Comment:
I noticed that `getOffsetWithDefault()` function is exported in `relay-js`.
It makes sense to export this to give user access to offset value from a connection string, similar to `CursorToOffset()`

Related commit with additional comments:
https://github.com/graphql/graphql-relay-js/commit/8f0fc3b2e848d328a98fa871aac7e84e88ab01a3#diff-fdeefc1c54f1bbbe87f2c14ef091b5b9